### PR TITLE
Add becklyn/assets-bundle

### PR DESCRIPTION
**Pull request by apfelbox**
| Q             | A
| ------------- | ---
| License       | MIT

Adding the recipe for https://packagist.org/packages/becklyn/assets-bundle

### CODE REVIEW

**Comment by apfelbox** [comment]
This one should be ready. It is supposed to fail on ^3.4, as we only support 4.2+

**Comment by apfelbox** [comment]
Sorry for bothering you, but is there something missing here? @fabpot @Nyholm

File: becklyn/assets-bundle/2.4/config/routes/becklyn_assets.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+_import.becklyn_assets:
+    resource: .
+    type: 'becklyn-assets'
```
Comment (Nyholm) [inline becklyn/assets-bundle/2.4/config/routes/becklyn_assets.yaml]: I dont quite understand this. Would you elaborate?

Comment (apfelbox) [reply becklyn/assets-bundle/2.4/config/routes/becklyn_assets.yaml]: Sure!

That is the matching configuration for our route loader: https://github.com/Becklyn/AssetsBundle/blob/master/src/RouteLoader/AssetsRouteLoader.php

The path of where the assets in your app should be at is configurable in the bundle.

This is required as the assets bundle can either provide the assets directly via a controller (there we need the path to configure a route dynamically – which is the link above and why this custom route loader is required) or we can statically dump the assets and serve them directly via the web server (in prod – here we need the path to know where to write the files to).

tl;dr as we have a configurable part in a route, we create the route dynamically.

Comment (Nyholm) [reply becklyn/assets-bundle/2.4/config/routes/becklyn_assets.yaml]: Cool. Thank you
